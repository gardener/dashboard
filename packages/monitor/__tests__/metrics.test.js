//
// SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
//
// SPDX-License-Identifier: Apache-2.0
//

'use strict'

const metrics = require('../lib/metrics')

describe('metrics', () => {
  it('should create and export the application defined metrics', () => {
    expect(Object.keys(metrics)).toMatchSnapshot()
  })
})
