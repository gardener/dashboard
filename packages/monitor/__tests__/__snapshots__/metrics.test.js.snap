// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`metrics should create and export the application defined metrics 1`] = `
[
  "connectionsCount",
  "connectionsTotal",
  "responseTime",
]
`;
